scalyr_agent/monitor_utils/k8s.py

```diff
@@ -877,6 +884,80 @@ def process_object(self, k8s, obj, query_options=None):
         raise NotImplementedError("process_object not implemented for _K8sProcessor")
 
 
+class NamespaceInfo():
+
+    @staticmethod
+    def __flatten_dict(dict_to_flatten):
+        flattened = []
+        for key in sorted(dict_to_flatten.keys()):
+            flattened.append(key)
+            flattened.append(six.text_type(dict_to_flatten[key]))
+
+        return "".join(flattened)
+    def __init__(self, name, uid, labels, annotations, spec):
+        self.name = name
+        self.namespace = name
```

Comment:
Why have two attributes for name(space)?

Reply:
Both attributes are needed elsewhere in our k8s library, which expects namespaced objects. This was the easiest way to ensure compatibility and no to break things.